.github/workflows/connector-tests.yml

```diff
@@ -74,6 +74,8 @@ jobs:
             cdk_extra: n/a
           - connector: source-google-drive
             cdk_extra: file-based
+          - connector: source-google-ads
+            cdk_extra: n/a
```

Comment:
The version on master is failing due to other unrelated issues (undeclared dependency and being a major CDK version behind). I'll remove this, since this PR can't merge without the other - and there's a workaround anyway for the Google Ads PR, which is to replace 'exception' with 'failed' in the `acceptance-test-config.yaml` file.